x/dex/keeper/msgserver/msg_server_unregister_contract.go

```diff
@@ -20,10 +26,7 @@ func (k msgServer) UnregisterContract(goCtx context.Context, msg *types.MsgUnreg
 		return nil, sdkerrors.ErrUnauthorized
 	}
 	// refund remaining rent to the creator
-	creatorAddr, err := sdk.AccAddressFromBech32(contract.Creator)
-	if err != nil {
-		return nil, err
-	}
+	creatorAddr, _ := sdk.AccAddressFromBech32(contract.Creator)
```

Comment:
should we still handle err?

Reply:
no we can assume the creatorAddr must be a valid bech32 address (as all msgserver has validated any request's address). This is also aligned with how other cosmos native modules convert bech32 address